### PR TITLE
[DOCS] Button to view indicators added to Threat Intelligence card 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2775.

Previews:
- [Overview Dashboard | Threat Intelligence](https://security-docs_2807.docs-preview.app.elstc.co/guide/en/security/master/overview-dashboard.html#_threat_intelligence)
- [Enable threat intelligence integrations](https://security-docs_2807.docs-preview.app.elstc.co/guide/en/security/master/es-threat-intel-integrations.html)

### CODE REVIEW

**Review by PhilippeOberti — APPROVED** [review]
LGTM!

File: docs/dashboards/overview-dashboard.asciidoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ image::images/events-count.png[Host and network events on the Overview dashboard
 
 The Threat Intelligence view on the Overview dashboard provides streamlined threat intelligence data for threat detection and matching.
 
-The view shows the total number of ingested threat indicators, enabled threat intelligence sources, and ingested threat indicators per source. To visualize the ingested threat indicator data, click the *Source* link for a threat intelligence source.
+The view shows the total number of ingested threat indicators, enabled threat intelligence sources, and ingested threat indicators per source. To learn more about the ingested indicator data, *View indicators*.
```
Comment (benironside) [inline docs/dashboards/overview-dashboard.asciidoc:37]:
```suggestion
The view shows the total number of ingested threat indicators, enabled threat intelligence sources, and ingested threat indicators per source. To learn more about the ingested indicator data, click *View indicators*.
```

**Review by benironside — APPROVED** [review]
LGTM, left one optional line edit

**Review by joepeeples — COMMENTED** [review]
+1 @benironside's edit, then LGTM!

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**